src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1251,7 +1254,9 @@ public static void applySslContext()
         {
             SSLFactory.validateSslContext("Internode messaging", conf.server_encryption_options, REQUIRED, true);
             SSLFactory.validateSslContext("Native transport", conf.client_encryption_options, conf.client_encryption_options.getClientAuth(), true);
+            SSLFactory.validateSslContext("JMX transport", conf.jmx_encryption_options, conf.jmx_encryption_options.getClientAuth(), true);
```

Comment:
@maulin-vasavada how do you know that `conf.jmx_encryption_options` is not `null` here?